packages/data/package.json

```diff
@@ -32,6 +32,7 @@
 		"equivalent-key-map": "^0.2.2",
 		"is-promise": "^2.1.0",
 		"lodash": "^4.17.14",
+		"memize": "^1.0.5",
```

Comment:
Did you use `lerna` to add this?

I ask because the root dep and lockfiles are not updated.

Reply:
No, I did not use Lerna, but I already updated the lock file.